src/GHC/Util.hs

```diff
@@ -38,17 +38,22 @@ import GHC.Util.Unify
 import qualified Language.Haskell.GhclibParserEx.Parse as GhclibParserEx
 import Language.Haskell.GhclibParserEx.GHC.Driver.Session (parsePragmasIntoDynFlags)
 
-import HsSyn
+import GHC.Hs
 import Lexer
 import SrcLoc
 import DynFlags
 import FastString
+import RdrHsSyn
 
 import System.FilePath
 import Language.Preprocessor.Unlit
 
 parseExpGhcLib :: String -> DynFlags -> ParseResult (LHsExpr GhcPs)
-parseExpGhcLib = GhclibParserEx.parseExpression
+parseExpGhcLib s flags =
```

Comment:
Why does this need to change? Do we not have parseExpression in ghc-lib-parser-ex?

Reply:
We do, but at this time, `parseExpression` its definition is this:
```
#if defined (GHCLIB_API_811) || defined (GHCLIB_API_810)
parseExpression :: String -> DynFlags -> ParseResult RdrHsSyn.ECP
#else
parseExpression :: String -> DynFlags -> ParseResult (LHsExpr GhcPs)
#endif
parseExpression = parse Parser.parseExpression
```
I thought when the dust settles I'll fix it up there so that code won't feature in hlint.

Reply:
Makes a lot of sense.

Reply:
Raised an issue for the next release https://github.com/shayne-fletcher/ghc-lib-parser-ex/issues/37